### PR TITLE
RavenDB-14847 Allow to compact database only (with no indexes) from t…

### PR DESCRIPTION
**Pull request by Danielle9897**
…he studio

### CODE REVIEW

File: src/Raven.Studio/typescript/commands/database/index/getIndexNamesCommand.ts [hunk 1]
```diff
@@ -14,7 +14,8 @@ class getIndexNamesCommand extends commandBase {
             pageSize: 1024
         };
         const url = endpoints.databases.index.indexes;
-        return this.query(url, args, this.db, (x: resultsDto<string>) => x.Results);
+        return this.query(url, args, this.db, (x: resultsDto<string>) => x.Results)
```
Comment (ml054) [inline src/Raven.Studio/typescript/commands/database/index/getIndexNamesCommand.ts:17]: please remove this

File: src/Raven.Studio/typescript/viewmodels/resources/compactDatabaseDialog.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
+import dialogViewModelBase = require("viewmodels/dialogViewModelBase");
+import databasesManager = require("common/shell/databasesManager");
+import notificationCenter = require("common/notifications/notificationCenter");
+import databaseInfo = require("models/resources/info/databaseInfo");
+import getIndexNamesCommand = require("commands/database/index/getIndexNamesCommand");
+import compactDatabaseCommand = require("commands/resources/compactDatabaseCommand");
+import genUtils = require("../../common/generalUtils");
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/resources/compactDatabaseDialog.ts]: update import

File: src/Raven.Studio/wwwroot/Content/css/pages/databases.less [hunk 3]
```diff
@@ -330,3 +330,9 @@
 #restorePointSpinner {
     margin-left: 10px;
 }
+
+#compactDatabaseDialog {
+    .limit-height {
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/Content/css/pages/databases.less:335]: I think we have already class for that: max-height so somethign like that ?

File: src/Raven.Studio/typescript/viewmodels/resources/compactDatabaseDialog.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/resources/compactDatabaseDialog.ts]: done

File: src/Raven.Studio/wwwroot/Content/css/pages/databases.less (hunk 3 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/Content/css/pages/databases.less:335]: There is an existing limit-height class with 250px 
```
.limit-height {
    max-height: 250px;
    overflow-y: auto;
}
```

Comment (ml054) [reply src/Raven.Studio/wwwroot/Content/css/pages/databases.less:335]: ok

**Review by ml054 — APPROVED** [review]

**Pull request merged**